_data/events/datasaturday0012.yml

```diff
@@ -0,0 +1,26 @@
+name: "Data Saturday #12 - Denver"
+date: 2021-09-09 08:00:00 -0700
+virtual: true
+description:  'Data Saturday #12 Denver is a free one-day event for professionals who use the Microsoft data platform. This community event offer content across data management, cloud and hybrid architecture, analytics, business intelligence, AI, and more.
```

Comment:
We need double quotes for description as everything after the # will be a comment and I don't think it will see the end single quote